src/main/kotlin/no/nav/familie/ef/sak/api/oppgave/FinnOppgaveRequestDto.kt

```diff
@@ -0,0 +1,44 @@
+package no.nav.familie.ef.sak.api.oppgave
+
+import no.nav.familie.kontrakter.felles.oppgave.Behandlingstema
+import no.nav.familie.kontrakter.felles.oppgave.FinnOppgaveRequest
+import no.nav.familie.kontrakter.felles.oppgave.Oppgavetype
+import no.nav.familie.kontrakter.felles.oppgave.Tema
+import java.time.LocalDate
+import java.time.LocalDateTime
+
+data class FinnOppgaveRequestDto(
```

Comment:
Kan dere ikke bruke denne? https://github.com/navikt/familie-kontrakter/blob/master/felles/src/main/kotlin/no/nav/familie/kontrakter/felles/oppgave/FinnOppgaveRequest.kt

Reply:
Ikke nødvendigvis. Tror vi skal "forenkle" input hvis det er meningen. Nå er det "hacks" i BA for å sette fom/tom og klokkeslett i JS-kode uten at det nødvendigvis gir verdi. Da kan vi heller forenkle hva som sendes inn og håndtere det her.

Reply:
Det mappes om til `FinnOppgaveRequest` nedenfor. :)

Reply:
Jo jeg tror vel att vi kan bruke den. Det ende vi gjør med denne er å sette tema når vi kaller `tilFinnOppgaveRequest`

Reply:
Ble noen andre endringer her som gjør at vi ikke ønsker å bruke kontrakter-modellen direkte